### DISCUSSION TITLE
Welcome to topola-viewer Discussions!

### DISCUSSION THREAD

**Discussion by PeWu** [PeWu/topola-viewer#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ludvig55** [comment]
Heló! Sajnos sok képzettségem nincsen a számítástechnikában de a családfámat szeretném "ábrázolni" NEM SZÖGLETES FORMÁBAN"(nem hagyományos ábrában). Kb 320 név van de ha sikerölni csak 4 vagy 5 generációt ábrázolni az is megfelelő lenne.
Megnyerte a tetszésemet https://pewu.github.io/topola/ linken a FANCY TREE ábrázolás. Ezt hogy lehet elérni? Ebben kérnék segítséget. Family search programból nyertem ki a GEDCOM fájlt és ezt alakítanám át. Mit kell csinálni és mive, hogy FA szerkezetű legyen a kép?
Üdvözlettel Lajos
![undefined - Imgur](https://user-images.githubusercontent.com/116481724/231776347-a1603c00-322c-46a4-bf54-a3cf5d36c218.jpg)

alias ludvig55
